### DISCUSSION TITLE
Is there a Wiki somewhere?

### DISCUSSION THREAD

**Discussion by stefonarch** [lxqt/lxqt#1873] [Q&A] [answered]

**Comment by sirviot** [comment, marked as answer]
Is the wiki in the GitHub OK? [https://github.com/lxqt/lxqt/wiki](https://github.com/lxqt/lxqt/wiki)

**Reply by tsujan** [reply]
I think @stefonarch made this as a test. Otherwise, yes, our Wiki is https://github.com/lxqt/lxqt/wiki

**Comment by stefonarch** [comment]
Yes, it was for testing, I see that it works voting the answers. 
But lxqt/lxqt is the only repo with a link to the wiki, so sometimes I looked for the link and didn't find it.

**Reply by tsujan** [reply]
I think GitHub's Discussions section is better than the old forum. No need to an extra sign-in; everything is done through GitHub. Its categories are enough (Ideas, Q&A, "Show and tell") but, if needed, we could add more.
